circle.yml

```diff
@@ -70,6 +66,10 @@ test:
     - gvm use stable && make lint:
         pwd: $BASE_STABLE
 
+  # Ensure validation of dependencies
+    - gvm use stable && if test -n "`git diff --stat=1000 master | grep -Ei \"vendor|godeps\"`"; then make dep-validate; fi:
+        pwd: $BASE_STABLE
+
```

Comment:
When this command fails it blows away the `vendor` causing all the other scripts to fail in confusing ways. I put it at the end so a failure would not cascade as badly. WDYT?